### DISCUSSION TITLE
Will adjusted DT and HT scores get a recalculation after they will be 'ranked'?

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29432] [Q&A] [answered]
So, if adujusted speed scores get ranked, will the previous scores with adjusted speed (like DT 2X) also get a recalculation so they will also give the user the pp they "missed out" on?

**Comment by peppy** [comment, marked as answer]
Yes

**Reply by bdach** [reply]
Just so that we're on the same page here - you're aware that this will require extra intervention on our behalf, yes? As in, locating the plays with the mods affected, setting `ranked = 1` for them in the database, and _then_ doing a pp recalc?

We have not previously done this for any mod that did not start off ranked when lazer got pp but became ranked later.

**Reply by Tenexxt** [reply]
Yes, thats what i meant, my bad for describing it wrong.

**Reply by peppy** [reply]
> Just so that we're on the same page here - you're aware that this will require extra intervention on our behalf, yes? 

If the scores aren't currently ranked then we might have to reconsider. Depends on the viability of locating which scores we consider to be valid-for-ranking.
